README.md

```diff
@@ -72,9 +72,7 @@ our [rules repository](https://jira.sonarsource.com/projects/RSPEC).
 
 Note: Our CI does not get automatically triggered on the PRs from external contributors.
 A member of our team will review the code and trigger the CI on demand by adding a comment on the PR (see [Azure Pipelines Comment triggers docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#comment-triggers)):
-- `/azp run Sonar.Net.External` - unit and integration tests for the analyzer (optional; can be done before the review)
-- `/azp run Sonar.Net` - full pipeline, including plugin tests and promotion (must be done before merging)
-
+- `/azp run Sonar.Net` - It will run the full pipeline, including plugin tests and promotion (must be done before merging)

```

Comment:
We can remove this part, GitHub checks and our daily work makes sure we will not forget
```suggestion
- `/azp run Sonar.Net` - It will run the full pipeline, including plugin tests and promotion

```